app/src/main/java/com/veniosg/dir/android/dialog/DetailsDialog.java

```diff
@@ -143,7 +143,7 @@ protected void onPostExecute(String result) {
 
         @Override
         protected String doInBackground(Void... params) {
-            return mFileHolder.getFormattedSize(getActivity(), true);
+            return mFileHolder.getSizeInfo(getActivity(), true);
```

Comment:
AFAIK this will break the details Dialog. We should still use `getFormattedSize(ctx, true)` here.